src/cli-util.js

```diff
@@ -299,7 +299,14 @@ function eachFilename(argv, patterns, callback) {
       return;
     }
     ignorer
-      .filter(filePaths)
+      .filter(
+        filePaths.map(
+          filePath =>
+            filePath.startsWith(process.cwd())
```

Comment:
Use `path.isAbsolute()` instead.

And I'd recommend the change to be at `globby.sync()` since it's kind of normalization:

```js
const filePaths =  globby.sync(...).map(...);
```